spring-data-relational/src/test/java/org/springframework/data/relational/core/sql/render/SelectRendererUnitTests.java

```diff
@@ -349,4 +349,23 @@ public void shouldRenderWithRenderContext() {
 		assertThat(rendered).isEqualTo(
 				"SELECT COUNT(\"my_table\".*) AS counter, \"my_table\".\"reserved_keyword\" FROM \"my_table\" JOIN \"join_table\" ON \"my_table\".source = \"join_table\".target");
 	}
+
+	@Test // DATAJDBC-353
+	public void shouldRenderWithColumnAlias() {
+		Table a = SQL.table("a").as("a_alias"); Column aColumn = a.column("a_col").as("a_col_alias");
+		Table b = SQL.table("b").as("b_alias"); Column bColumn = b.column("a_col").as("b_col_alias");
```

Comment:
For the column name we should use `"b_col"` makes it easier to understand. 
This needs to change in the assert statement as well.